src/net/sourceforge/kolmafia/utilities/FileUtilities.java

```diff
@@ -355,6 +355,9 @@ private static String localImageName(final String filename) {
     }
     String images = ".com";
     int index = filename.lastIndexOf(images);
+    if (index == -1) {
```

Comment:
This is fine as is.  But it could be futureproofed by using the image server enumeration to derive candidate strings rather than hardwire com and net.  Of course if there are ever different images on different servers and it matters which one is returned then this, and the derivation, would need to be documented or reimplemented :-)